pkg/vmimage/data/script.sh

```diff
@@ -164,7 +172,29 @@ yum -y install \
     tree \
     find \
     WALinuxAgent \
-    yum-utils
+    yum-utils \
+    azure-mdsd \
+    azure-security \
+    azsec-monitor \
+    td-agent
+
+# td-agent needs access to the journal
+usermod -a -G systemd-journal td-agent
```

Comment:
```suggestion
adduser td-agent systemd-journal
```

`usermod` is pretty low-level and the syntax is sort of arcane, I'd prefer using `adduser` unless there's a compelling reason not to?